doc: Add Ubidots integration guide (#2835)
TheThingsNetwork/lorawan-stack: doc/content/integrations/webhooks/cloud-integrations/ubidots/tts-setup/_index.md

For more information on message payload formatters, see [Payload Formatters]({{< ref "/integrations/payload-formatters" >}}).

For this guide, we will use the following example JavaScript payload formatter, which converts a byte encoded temperature and humidity to human readable fields.

```
function Decoder(bytes, fport) {
  var decoded = {};

  var temp = (bytes[0] << 8) | bytes[1];
  var hum = (bytes[2] << 8) | bytes[3];
  
  decoded.temperature = temp / 100;
  decoded.humidity = hum / 100;

  return decoded;
}
```

You should adjust the decoded based on the payload that your device is sending.

## Creating a Webhook Integration

>Note: this section follows the [HTTP Webhooks]({{< ref "/integrations/webhooks" >}}) guide. 

Fill in the **Webhook ID** field and choose **JSON** for **Webhook format**. 

Create a **Content-type** header with **application/json** value.

Paste the copied URL of your Ubidots function to **Base URL** field.

Check the uplink message type and select **Add Webhook** to finish creating an integration. 

> {{< figure src="ubidots-webhook-creation.png" alt="Ubidots webhook" >}}

Reviewer: You sure this is correct? 

The Ubidots guy added the path (after `/prv/`) in the Message type during the virtual conference. I suppose this doesn't matter, but pls double check.
![Screenshot 2020-07-14 at 14 02 04](https://user-images.githubusercontent.com/17290639/87423465-a3362b00-c5da-11ea-99d3-a85b9345087b.png)

Author: This works correctly, you can do it both ways

Once you have created the integration, navigate to **Devices** tab in Ubidots dashboard and select **Devices**. 

> In the list of devices, you will be able to find your device, since it is being automatically added. Click on the device and you will see automatically added variables with their latest values.

Reviewer 2: ```suggestion
You should see your device listed, as it is automatically added when an uplink is received. Click the device to see variables and their latest values.
```
